changelog.txt

```diff
@@ -0,0 +1,12 @@
+Upcoming Version
```

Comment:
This looks good! Though I would rename the file to `CHANGELOG.md` and add `#` or `##` to this and to `0.10.1` below